internal/endtoend/testdata/star_expansion_reserved/mysql/go/query.sql.go

```diff
@@ -0,0 +1,36 @@
+// Code generated by sqlc. DO NOT EDIT.
+// source: query.sql
+
+package querytest
+
+import (
+	"context"
+	"database/sql"
+)
+
+const starExpansionReserved = `-- name: StarExpansionReserved :many
+SELECT ` + "`" + `group` + "`" + ` FROM foo
```

Comment:
It's gross, but it works 100% of the time and doesn't require a call to `strings.Replace`